rootfs/init.go

```diff
@@ -69,14 +69,6 @@ func createInitLayer(ctx context.Context, parent, initName string, initFn func(s
 	if err != nil {
 		return "", err
 	}
-	defer func() {
-		if err != nil {
-			// TODO: once implemented uncomment
-			//if rerr := snapshotter.Remove(ctx, td); rerr != nil {
-			//	log.G(ctx).Errorf("Failed to remove snapshot %s: %v", td, merr)
-			//}
```

Comment:
`snapshotter.Remove()` has been implemented, but this code doesn't work anymore (`merr` is defined later).  If this is still necessary I can open a new issue to restore this.

Reply:
@dmcgowan is this needed with the GC now in place?  I don't think so but you would know better than I.

Reply:
@dmcgowan responded on slack that this is still necessary. I'll add another commit which restores this section.

Reply:
restored this defer